### DISCUSSION TITLE
how to make it to push the button like Options or other interactive with the shown figure?

### DISCUSSION THREAD

**Discussion by hjhj111** [alandefreitas/matplotplusplus#191] [Q&A]
how to make it to push the button like Options or other interactive with the shown figure?
the window is always no response, (the plot is shown).

**Comment by KnIfER** [comment]
> the window is always no response

强制用--persist 参数启动 gnuplot（注释掉 matplot::wait();）

**Reply by hjhj111** [reply]
matplot::plot(data_, count_, ":","--persist");
Does it like this?  It does not work

**Comment by hjhj111** [comment]
static constexpr bool windows_should_persist_by_default = true;
it does not work too

**Comment by hjhj111** [comment]
can you please give an example?
